### ISSUE TITLE
Adding Validations  

### ISSUE THREAD

**Issue by granthoff1107** [bchavez/Coinbase#50]
We should Add validations for certain things such as creating transactions.
When creating transactions amounts should be fixed at 8 decimal places, etc

**Comment by bchavez** [comment]
Does Coinbase give back an error if the number of decimal places is off?

I prefer not to have any (or very little) validation in the client and have all validation being done on their servers. Mostly for maintenance reasons, as their limits/validation-rules can change over time wont require us to change too.

It could be helpful to have the client throw before an HTTP request gets sent; however, if the consumer gets the same error result client-side as they do server-side, then I think it would just be duplicate validation work.

IMHO, we should just provide the minimum transport mechanism to get requests on-boarded to their server network to do work.

What other validation did you have in mind besides the fixed 8 decimal places?

**Comment by granthoff1107** [comment]
Good point, yeah I was getting an error when trying to send from the server.

I don't remember what it was but, but I think it was vague like incorrect
amount instead of,  the amount has too much precision.

It was a vague error so I figured we could save others the headache.

But it's not too important
